### ISSUE TITLE
Ability to configure API URL on a per-project basis

### ISSUE THREAD

**Issue by muety** [wakatime/wakatime-cli#569]
A user of [Wakapi](https://github.com/muety/wakapi) has expressed a wish for being able to configure the API URL on a per-project basis to use WakaTime alongside, for instance, a self-hosted Wakapi in parallel. Some projects could be tracked in one service and some in another. Suggestions was to have something analogously to the `.wakatime-project` file in a project's folder.

I know that this would be quite a non-trivial change, so I'd like to ask you whether you even consider this an option for WakaTime CLI.

**Comment by karolzlot** [comment]
I came up with alternative way to solve this problem:

Instead of choosing per project it would be enough to have all heartbeats send to both backends. In this case data in wakatime and wakapi would be duplicated and users would be able to use both services as they need. Now I think this solution is the best because it allows to use features of both backends in each project. (It can happen that one feature is available only on wakatime and another only on wakapi backend)

It would be best to solve it in `wakatime-cli` but alternatively it can be hacked on wakapi side: when wakapi receives hearbeat then it sends it also to wakatime. Another way to hack it is probably by some proxy-like solution which sends hearbeats to both backends.

**Comment by muety** [comment]
Wakapi already supports forwarding heartbeats to WakaTime. However, "splitting" them per project is not an option right now. Let's move this discussion out of here, though, as we're getting out of scope of wakatime-cli.

**Comment by karolzlot** [comment]
@muety 

Ok, moved.

**Comment by alanhamlett** [comment]
Closing now that conversion is at https://github.com/muety/wakapi/issues/82#issuecomment-955586107

**Comment by karolzlot** [comment]
@alanhamlett 
This issue is about configuring backend per project.
Please reopen if this could be possible in `wakatime-cli`.

**Comment by alanhamlett** [comment]
It's possible, but I don't think it's likely to be implemented. It makes more sense to use the existing duplication so both local and cloud show the same total time instead of having a fragmented picture of the day.

**Comment by karolzlot** [comment]
@alanhamlett Ok, I agree with this argument.
